package.json

```diff
@@ -34,26 +34,10 @@
       "Ammo": "readonly",
       "earcut": "readonly",
       "glslang": "readonly",
-      "GPUAdapter": "readonly",
-      "GPUBindGroup": "readonly",
-      "GPUBindGroupDescriptor": "readonly",
-      "GPUBindGroupLayout": "readonly",
-      "GPUBlendState": "readonly",
-      "GPUBuffer": "readonly",
       "GPUBufferUsage": "readonly",
```

Comment:
these 4 should be removed as well I think

Reply:
The linter complains if you remove those 4.